### PR TITLE
sys-kernel/dracut: add 059_p20231027

### PR DESCRIPTION
**Pull request by LaszloGombos**
Add a new version of dracut based on the latest git commit.
With this version upgrade I was able to remove 5 patches maintained in the tree.
Help is needed to test this upgrade.

It has been over 10 months since the last upstream dracut release. Upstream release generation is stalled, see - https://github.com/dracutdevs/dracut/issues/2431#issuecomment-1754305440

CC @thesamesam

### CODE REVIEW

File: sys-kernel/dracut/dracut-059_p20231027-r5.ebuild [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+	local myconf=(
+		--prefix="${EPREFIX}/usr"
+		--sysconfdir="${EPREFIX}/etc"
+		--bashcompletiondir="$(get_bashcompdir)"
+		--systemdsystemunitdir="$(systemd_get_systemunitdir)"
+	)
+
+	tc-export CC PKG_CONFIG
+
+	echo ./configure "${myconf[@]}"
```
Comment (mm1ke) [inline sys-kernel/dracut/dracut-059_p20231027-r5.ebuild:79]: This only prints the command which is going to be executed next, doesn't it?
Wouldn't a `einfo` fit better here? Something like:
``` suggestion
	einfo "./configure ${myconf[@]}"
```

**Comment by mm1ke** [comment]
BTW, could you also update the HOMEPAGE of the 9999 ebuild?

**Comment by thesamesam** [comment]
Thank you Laszlo! I've fixed up `S` as the layout of github snapshot tarballs is different to tags. Tested and everything seemed to work fine too.

**Comment by floppym** [comment]
Thanks for the bump.

**Pull request closed without merging**